cmd/wallet-web/src/components/Modal/Credential/Delete.vue

```diff
@@ -0,0 +1,152 @@
+<template>
```

Comment:
Inside `src/components/Modal` we should have `Modal.vue` component, which would provide the `teleport` functionality and would have a `slot` inside to insert content dynamically. The modal the delete a credential should probably be placed under `src/components/CredentialDetails/DeleteCredentialModal.vue` and would import `src/components/Modal/Modal.vue` to get the `teleport` functionality and then add it's unique content in the slot.

Reply:
Moved the modal as commented